npm-js/kalix-scripts/bin/kalix-scripts.js

```diff
@@ -76,14 +113,16 @@ const scriptHandlers = {
       'Invoking Kalix codegen',
       path.resolve(kalixScriptDir, 'bin', 'kalix-codegen-js.bin'),
       [
-        '--proto-source-dir',
-        protoSourceDir,
-        '--source-dir',
-        sourceDir,
-        '--test-source-dir',
-        testSourceDir,
-        '--generated-source-dir',
-        generatedSourceDir,
+        ...flag('--typescript', typescript),
+        ...option('--base-dir', baseDir),
+        ...option('--main-file', mainFile),
+        ...option('--descriptor-set-output-dir', descriptorSetOutputDir),
+        ...option('--descriptor-set-file', descriptorSetFile),
+        ...option('--proto-source-dir', protoSourceDir),
+        ...option('--source-dir', sourceDir),
+        ...option('--generated-source-dir', generatedSourceDir),
+        ...option('--test-source-dir', testSourceDir),
+        ...option('--integration-test-source-dir', integrationTestSourceDir),
```

Comment:
Support all of the possible options in the config. I think it could also be good to support a separate config json file. And the `kalix-scripts build` could be used for the testkit example project now. Will follow up on these separately, and any other codegen changes.